CONTRIBUTING.md

```diff
@@ -17,19 +16,14 @@ setup = {
 ```
 
 ## Creating a new lesson
-Inside the `stage/[stage_name]` folder (such as `stage/one`).  
-Make sure there is a mininum `__init__.py`.  
+Make sure there is a `__init__.py` inside the `stage/[stage_name]` folder (such as `stage/one`).
 
-
-And create new file with any name you like ,
-But file name show same as function name.  
+Create a new file with any name you like (it should however follow the same conventions as a function name). 
```

Comment:
Oops. the function name is no longer needs to follow file name.
It's now called `sanic_request`.

I think this paragraph can be remove.